wfl/fit/gap_simple.py

```diff
@@ -112,9 +112,9 @@ def run_gap_fit(fitting_configs, fitting_dict, stdout_file, gap_fit_exec="gap_fi
     if not fitting_configs_filename:
         fitting_configs_scratch_filename = fitting_configs.to_scratch_file('_GAP_fitting_configs.xyz')
         fitting_configs_filename = fitting_configs_scratch_filename
-
+    
     # kwargs overwrite the fitting_dict given
-    use_fitting_dict = dict(fitting_dict, atoms_filename=fitting_configs_filename, **kwargs)
+    use_fitting_dict = dict(fitting_dict, atoms_filename=str(fitting_configs_filename), **kwargs)
```

Comment:
If we do decide to go with always storing `Path` internally, maybe `dict_to_gap_fit_string()` is the place where it should convert from Path to str.

Reply:
Everything gets converted to string by `ase.io.extxyz.key_val_dict_to_str`, which doesn't recognise `Path` and omits the entry altogether. I'll see about suggesting a fix in ase.

Reply:
Oh, yes.  Probably not worth trying to get a change into ase.io, since we're planning to switch to an entirely different extxyz reader soon anyway.

Reply:
Ah `pathlib` is not a standard module, so adding a two-line check to `extxyz.py` would need to introduce a new dependency to ase - I don't think that's an option (?).  I'll add a function to `dict_to_gap_fit_string()`.

Reply:
didn't see your message above - ok!